### DISCUSSION TITLE
Reconciling too many transactions generates a too long URL, leads to error 414

### DISCUSSION THREAD

**Discussion by goebeler** [firefly-iii/firefly-iii#9286] [Firefly III]
I did not yet want to file a bug report, but it looks like selecting a lot of transactions and trying to save during reconcile leads to some url being generated that's actually too long. 

In my case nginx is throwing the error. I know that I can make it accept longer URLs and bigger headers but I think that's just masking the problem that firefly generates huge urls (endless entries of "&journals%5B%5D=4445&journals%5B%5D=4443&journals%5B%5D=4442...")

Maybe it would be possible to send the journal data (?) as payload of a put or patch request. So far I've not taken a look at the firefly internals.

**Comment by JC5** [comment]
Hey, thanks for starting a discussion. 

This issue has been reported many times, please check out the search function. It is on my list. In the meantime, reconcile smaller batches.
